dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/CpuInfo.java

```diff
@@ -62,16 +62,6 @@ private CpuInfo() {
             AVAILABLE_PROCESSORS = fileSystemAvailableProcessors;
         }
 
-        // initialize mbean required to get CPU usage info
-        osmx = ManagementFactory.getOperatingSystemMXBean();
-        mBeanServer = ManagementFactory.getPlatformMBeanServer();
-
-        try {
-            operatingSystemMbean = new ObjectName("java.lang", "type", "OperatingSystem");
-        } catch (MalformedObjectNameException e) {
-            FFDCFilter.processException(e, getClass().getName(), "CpuInfo<init>");
-        }
-
```

Comment:
I suggest that you do something like this:

```
OperatingSystemMXBean osmx = ManagementFactory.getOperatingSystemMXBean();
operatingSystemMbean = osmx == null ? null : osmx.getObjectName();
```
Get rid of osmx as a class field and just have operatingSystemMbean be final.

Reply:
Done